cupy/cuda/compiler.py

```diff
@@ -133,7 +133,7 @@ def _get_arch():
         # CUDA 10.1 / 10.2
         _nvrtc_max_compute_capability = '75'
     else:
-        # CUDA 11.0 / 11.1 / 11.2
+        # CUDA 11.0 / 11.1 / 11.2 / 11.3 / 11.4
         _nvrtc_max_compute_capability = '80'
```

Comment:
Could you allow 86 for CUDA 11.1+?

Reply:
Can we also add that of `87`? It seems that nvcc 11.4 also supports `compute_87/sm_87`.

Reply:
I think it's safer to keep `86` here (when the new GPU with CC = 90 is released users may hit this) as it's undocumented and CUDA examples don't use it.
https://github.com/NVIDIA/cuda-samples/search?l=Makefile&q=SMS